dictsource/en_rules

```diff
@@ -6915,7 +6915,6 @@ contin) u          ju:
         ψ          ps'aI_
         ω          'oUmIg@_
 ?3      ω          oUm'eIg@_
-        ο          'oUmIkr0n_
```

Comment:
Yes, because there is flat **ο** in 6903 line:
and accented **ό** is replaced to flat **ο** in line 56.